samcli/cli/cli_config_file.py

```diff
@@ -266,8 +267,15 @@ def save_command_line_args_to_config(
         # if param was passed via command line, save to file
         param_value = ctx.params.get(param_name, None)
         if param_value is None:
+            LOG.debug(f"Parameter {param_name} was not saved, as its value is None.")
             continue  # no value given, ignore
         config_file.put(cmd_names, "parameters", param_name, param_value, config_env_name)
+        saved_params.update({param_name: param_value})
+
+    LOG.debug(
```

Comment:
Should this be an info log instead of a debug? Mainly asking since this is something that the customer is actively doing (providing `--save-params`), so we might want to report that something happened from the SAM CLI side of things that doesn't require a `--debug` to find out.

Reply:
Yup, giving customers that feedback would definitely help, will make that change 👍